coderkevin/fresh-data-wp-plugin-example
# Add base react app

This adds the base react application code.

To Test:
1. Clone this repo under `wp-content/plugins` in a WP dev install.
2. `npm install`
3. `npm start`
4. Enable the plugin
5. Visit the "Fresh Data 🥕" admin menu item.
5. Verify you see "Hello, from React!" in large blue letters.
